daskml/utils.py

```diff
@@ -0,0 +1,18 @@
+import dask.dataframe as dd
+import dask.array as da
+
+
+def slice_columns(X, columns):
+    if isinstance(X, dd.DataFrame):
+        return X[columns or list(X.columns)]
```

Comment:
Maybe

```python
if columns is None:
    columns = X.columns
```

Otherwise this will fail for `columns=Index(['a', 'b'])`, since `bool(columns)` is ambiguous.